### DISCUSSION TITLE
Graphs not reactive?

### DISCUSSION THREAD

**Discussion by rachithaiyappa** [fonsp/Pluto.jl#1054] [Questions about using Pluto] [answered]
Hi there,
Thanks for Pluto (Its awesome)
Shouldn't the first plot also get updated when the edge of the graph is set to 1000? Maybe I've misunderstood reactivity? Apologies for posting an image. Can share code if that's the protocol here.

![image](https://user-images.githubusercontent.com/44749902/113469095-c038b580-9418-11eb-99cb-6be523b10814.png)

**Comment by fonsp** [comment]
What do you mean with:

> when the edge of the graph is set to 1000

remember that we don't know the context of what you are trying to do

**Reply by rachithaiyappa** [reply]
The function `update_edge_weight`, defined in the 6th cell, when called, updates the edge weight of the existing graph (initially with edge weights shown as labels `1.5, 2.5, 3.5`) to (`1000, 2.5, 3.5`). So when I call this function (in the 5th cell) it does this update to graph `g`. I was under the impression that the reactivity of pluto automatically updates the graph plot (`gplot`) in the 4th cell where graph `g` has been referenced. 

In other words, when `g` is changed, I expected the plot in the 4th cell to be reactive to this change and resemble what is shown in the last cell but this doesn't seem to happen? Was wondering if this is an issue with the reactivity of pluto?

**Comment by fonsp** [comment, marked as answer]
Take a look at #564 -- mutations are not reactive in Pluto, but you can define "dummy variables" to trigger re-evaluation.

**Reply by rachithaiyappa** [reply]
Thanks Fons. Thats what I did end up doing but just wanted to confirm that something isn't fishy. Hope Pluto keeps growing!

**Reply by fonsp** [reply]
Thankyou!
